website/src/_posts/2022-12-3.3.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "🎄 Uppy 3.1—3.3: Improved AWS S3 Multipart, Single File Mode" 
+date: 2022-12-23
+author: 
+  - aduh95
+  - arturi
+  - murderlon
+  - mifi
+image: "https://uppy.io/images/blog/3.1-3.3/single-file-mode.jpg"
+published: true
+---
+
+Three minor releases ago we’ve released Uppy 3.0. It’s time to give you an update on what we’ve been working on all this time.
+
+TL;DR: AWS S3 Multipart stability improvements, Single File Mode, more tests and bugfixes, new locales.
+
+## AWS S3 Multipart: improved performance on slow connections
+
+We’ve been doing a lot of tests to find out why some uploads with `@uppy/aws-s3-multipart` plugin failed. It turned out, simultaneously signing and uploading many chunks at once saturated the HTTP/1.1 connection limit, which lead to signatures expiring by the time Uppy started uploading the next chunk.
+
+We’ve refactored our internal queue mechanism to sign chunks right before we start uploading them to the remote server. When the upload is paused, the queue is emptied and re-populated again on resume. The default `limit` for this plugin has been set to `6`. See [PR #4187](https://github.com/transloadit/uppy/pull/4187) for more details.
+
+To improve things even further, the default S3 expiry has been lifted from `300` to `800` seconds ([PR #4206](https://github.com/transloadit/uppy/pull/4206)).
+
+@uppy/aws-s3-multipart: handle slow connections better (Antoine du Hamel / #4213)
+
+## Single File Mode
+
+Often times people only want to upload a single file (most commonly an image) via Uppy. But the Dashboard UI is tailored to handling many files at once, and it used to display the single image as a small, sad-looking block in the corner. This ends now! (Actually, it ended a few releases ago, figure of speech).
+
+<img src="/images/blog/3.1-3.3/single-file-mode.jpg" class="border" />
```

Comment:
nit: can we make a screenshot with the dashboard which contains a xmas related picture?